include/openenclave/internal/report.h

```diff
@@ -85,6 +85,12 @@ OE_STATIC_ASSERT(
     }
 #define X509_OID_FOR_QUOTE_STRING "1.2.840.113556.10.1.1"
 
+#define X509_OID_FOR_NEW_QUOTE_EXT                           \
```

Comment:
I didn't see this define is used anywhere in the code. Am I missing something?

Reply:
Same above - enable verifier scenario first.